test/cli_test.rb

```diff
@@ -15,12 +15,13 @@ def test_secrets_uses_secrets_path_to_find_ejson_file
     Dir.mktmpdir do |dir|
       File.write(config_path = "#{dir}/config.yml", build_record_store_config)
       File.write("#{dir}/#{EJSON_PUBLIC_KEY}", EJSON_PRIVATE_KEY)
-      File.write(secrets_ejson_path = "#{dir}/secrets.#{ENV['CI'] ? 'ci' : 'dev'}.ejson", secrets_ejson = """
-{
-  \"_public_key\": \"#{EJSON_PUBLIC_KEY}\",
-  \"secret\": \"password\"
-}
-""")
+      File.write(secrets_ejson_path = "#{dir}/secrets.#{ENV['CI'] ? 'ci' : 'dev'}.ejson", secrets_ejson = <<~EJSON
+        {
+          "_public_key": "#{EJSON_PUBLIC_KEY}",
+          "secret": "password"
+        }
+      EJSON
+      )
```

Comment:
I think this would be a lot more readable split into multiple lines assigning vars and then using

Reply:
This feels better just looking at it 😌